### ISSUE TITLE
Facilitate piping-in spreadsheet-content from STDIN

### ISSUE THREAD

**Issue by Tableator** [roo-rb/roo#343]
Roo either demands that a file-extension conforms to the obvious spreadsheet-content or can be forced to “_assume_” a certain file-extension.

Apart from the fact, that Ruby is also used on operating-systems where the file-extension is not at all of the same significance as on Microsoft®-systems, this insistence on a file-extension complicates the development of pipe-in functionality. 

**Example**: I want to pipe-in E-Mail attachments to text-mode spreadsheet viewer. It is very easy to verify the mime-type or other properties of the incoming data and create the pertinent Roo-instance. However, I am obliged to direct the data to a file, first, then give it a file-extension or tell Roo what the file-extension should be, despite the fact, that **I know perfectly well the type of spreadsheet**, that I am handling!

In short: **All works.** **But it is a little cumbersome for reasons that are not very easy to accept**.

I suggest (or rather “imagine”) that alternatively to an imposed file-extension, we should have the liberty to impose a Roo-Instance by just naming a mime-type for any data streaming in... 

The risk must be ours and we will not sue you. Rest assured. ;)
